.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,49 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: The name of the Tarantool build artifact
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    strategy:
+      matrix:
+        metrics-version: [ "0.12.0" ]
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone the crud module
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.repository_owner }}/crud
+
+      - name: Download the Tarantool build artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.artifact_name }}
+
+      - name: Install Tarantool
```

Comment:
Please add this comment before this step.

```
# Now we're lucky: all dependencies are already installed. Check package
# dependencies when migrating to other OS version.
```

Reply:
@ylobankov what does it mean? How are they already installed if we install them in the next step?

Reply:
```suggestion
      # All dependencies for tarantool are already installed in Ubuntu 20.04.
      # Check package dependencies when migrating to other OS version.
      - name: Install Tarantool
```

Reply:
Fixed.